src/plugins/Dashboard/index.js

```diff
@@ -267,28 +273,31 @@ module.exports = class DashboardUI extends Plugin {
       showModalTrigger.forEach(trigger => trigger.removeEventListener('click', this.openModal))
     }
 
-    this.removeDragDropListener()
-
     if (!this.opts.inline) {
       document.removeEventListener('keydown', this.onKeydown)
     }
-  }
 
-  actions () {
-    this.core.on('dashboard:file-card', this.handleFileCard)
+    this.removeDragDropListener()
 
-    window.addEventListener('resize', this.updateDashboardElWidth)
+    this.uppy.off('dashboard:file-card', this.handleFileCard)
+    this.uppy.off('file-added', this.addDashboardMetaToNewFile)
+
+    window.removeEventListener('resize', this.updateDashboardElWidth)
   }
 
-  removeActions () {
-    this.core.off('dashboard:file-card', this.handleFileCard)
+  addDashboardMetaToNewFile (file) {
```

Comment:
I kinda feel like the dashboard as a UI plugin shouldn't be setting initial file metadata, maybe the `metaFields` option should only contain the placeholder and name data, and not a default value. then the `meta: {}` option to Core can be used to specify defaults. what's your thoughts on that?